src/App.js

```diff
@@ -15,7 +16,9 @@ export default function App() {
   //.catch(error => console.log(error))
 
   function createCards(characters) {
-    const cards = characters.map(character => Card(character.name))
+    const cards = characters.map(character =>
```

Comment:
You could already use de-structuring assignment here!
```suggestion
    const cards = characters.map(({ name, image }) =>
```